.github/workflows/release.yml

```diff
@@ -63,3 +63,31 @@ jobs:
           publish_dir: ./dist
           destination_dir: playground
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+  release-binaries:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            artifact_name: boa
+            asset_name: boa-linux-amd64
+          - os: windows-latest
+            artifact_name: boa.exe
+            asset_name: boa-windows-amd64
+          - os: macos-latest
+            artifact_name: boa
+            asset_name: boa-macos-amd64
```

Comment:
It'll probably be better to also set the rust target on compilation, just in case.
Example: https://github.com/crate-ci/cargo-release/blob/db00d8a078799d81546416031e2944e775be62e9/.github/workflows/post-release.yml#L55-L74

Reply:
Also, that release script also uses the target name to name the asset itself, which is arguably better for labeling

Reply:
I've made adjustments to this, the asset name may be confusing with the "boa" name in it, so ive slightly adjusted it